src/deprecated/index.ts

```diff
@@ -0,0 +1,30 @@
+/** This is the place where we keep components that are deprecated.
+ *  We don't recommend using it in production.
+ *  If you already use them, please move to the suggested alternative components
+ *
+ *  But, they are published on npm and you can import them.
+ *  example: import {FormGroup} from '@primer/react/deprecated
+ */
+
+export {default as BorderBox} from '../BorderBox'
+export type {BorderBoxProps} from '../BorderBox'
+export {default as Flex} from '../Flex'
+export type {FlexProps} from '../Flex'
+export {default as Grid} from '../Grid'
+export type {GridProps} from '../Grid'
+export {default as Position, Absolute, Fixed, Relative, Sticky} from '../Position'
+export type {PositionProps, AbsoluteProps, FixedProps, RelativeProps, StickyProps} from '../Position'
+export {default as Dialog} from '../Dialog'
+export type {DialogProps, DialogHeaderProps} from '../Dialog'
+export {ConfirmationDialog} from '../Dialog/ConfirmationDialog'
+export {default as Dropdown} from '../Dropdown'
+export type {
+  DropdownProps,
+  DropdownCaretProps,
+  DropdownButtonProps,
+  DropdownItemProps,
+  DropdownMenuProps
+} from '../Dropdown'
+export {default as FormGroup} from '../FormGroup'
+export type {FormGroupProps, FormGroupLabelProps} from '../FormGroup'
+export {default as SelectMenu} from '../SelectMenu'
```

Comment:
Think you're missing the props for `SelectMenu`, which are still in root index.ts

Reply:
Done!